### DISCUSSION TITLE
WSL2 The exception s (0x, possibly caused by renegate registry entries. Can't install or start.

### DISCUSSION THREAD

**Discussion by abalter** [microsoft/WSL#6652] [General]
# Environment

```none
Windows build number: [run `cmd.exe /c ver`] Microsoft Windows [Version 10.0.19041.804]
Your Distribution version: [On Debian or Ubuntu run `lsb_release -r` in WSL]
Whether the issue is on WSL 2 and/or WSL 1: [run `cat /proc/version` in WSL] WSL2
```

# Steps to reproduce

1. Uninstall Ubuntu
1. Turn off all virtualization windows features
1. Turn off WSL
1. Restart
1. Turn all features back on
1. Install WSL
    - From store
    - Importing previously exported distro
    - Downloading the .appx file from microsoft.
    - Using `lxrunoffline  
1. Get error

**WSL logs**: 

#  Expected behavior

Successfully move my wsl2 instance to a new computer

# Actual behavior

Error shown above. 

```
C:\Users\eacriadmin>wsl --import Ubuntu C:\wsl_ubuntu D:\spectre_wsl_export
{Application Error}
The exception s (0x
```

## Additional notes:
I was trying to move a wsl2 instance from another computer. I tried two methods. `wsl --import` on the export from the other machine. And `lxrunoffline import` directly on the .vhdx file. Both created a running wsl2 instance but with two problems. 1) It started up as root, and I had to create a `/etc/wsl.conf` file to start up as my user. 2) No networking capability.

As I tried, again and again, doing things in different orders, eventually I got to the current state where everything I do results in the error shown.

At some point, I started to see an additional LsxxManager registry item with some hex numbers. I deleted it and it would come back. 

![image](https://user-images.githubusercontent.com/5349876/110257400-9774eb80-7f52-11eb-8079-4fb055d1ac6a.png)

It also shows up in services

![image](https://user-images.githubusercontent.com/5349876/110257361-6eecf180-7f52-11eb-9fbe-905b3f37ef80.png)

I will be glad to upload some log files. Since there are many different operations I'm using, please let me know which would be the most useful to document.

**Comment by abalter** [comment]
I followed the steps [here](https://github.com/microsoft/WSL/issues/5508#issuecomment-743733155) explicitly and I think I'm up and running now. But only with a fresh install. I'll have to move all of my stuff over and reinstall all packages. But at least I have networking.

**Comment by cocktail828** [comment]
any resolution??? I meet the issue today.......
